critiquebrainz/db/users.py

```diff
@@ -532,6 +532,51 @@ def get_reviews(user_id, from_date='1-1-1970'):
     return [dict(row) for row in rows]
 
 
+def get_comments(user_id, from_date='1-1-1970'):
+    """Get comments on reviews by a user from a specified time.
+
+    Args:
+        user_id(uuid): ID of the user.
+        from_date(datetime): Date from which comments on reviews submitted by user are to be returned.
```

Comment:
the docstring says this is a datetime object, but you pass a string as the default.